examples/Joints/Ingen/slidingjoint.py

```diff
@@ -0,0 +1,21 @@
+from sympy import Symbol
```

Comment:
Yes, its a mass and bob example with gravity in the same direction as DoF.

Sorry, I will take care of that next time.